ext/phar/tests/gh18953.phpt

```diff
@@ -0,0 +1,45 @@
+--TEST--
+Phar: Stream double free
+--EXTENSIONS--
+phar
+--INI--
+phar.readonly=0
+--ENV--
+USE_ZEND_ALLOC=0
+--FILE--
+<?php
+
+declare(strict_types=1);
+
+require __DIR__ . '/gh18953/autoload.inc';
+
+// cleaning
+@unlink("gh18953.phar");
+@unlink("gh18953.phar.gz");
```

Comment:
This shouldn't be needed because of the CLEAN section

Reply:
A CLEAN section always runs, if it doesn't there are bigger problems with the test runner. Please remove it as asked previously.